snippets/csharp/VS_Snippets_CLR/CatchException/CS/catchexception2.cs

```diff
@@ -10,7 +10,7 @@ public static void Main()
         {
             StreamReader sr = File.OpenText("data.txt");
```

Comment:
Since StreamReader is disposable, it should be wrapped in a using statement. The call to Close can then be removed.